spec/requests/api/journalist_can_create_an_article_spec.rb

```diff
@@ -0,0 +1,90 @@
+RSpec.describe 'POST/api/articles', type: :request do
+  let!(:category) { create(:category, label: 'global_politics') }
+  let!(:category_2) { create(:category, label: 'global_politics') }
+
+  describe 'successfully creates an article' do
+    before do
+      post '/api/articles',
+           params: {
+             article: {
+               title: 'A title',
+               lead: 'And a lead',
+               body: 'With a body',
+               category_id: category.id
```

Comment:
How will you get the id of the category in the client? Would it be an easier solution for you in the client to sned in the label of the category and then here convert that by finding the category with that label and pass in that id when you save the article?
Just so you have thought this through and know how to deal with it in the client. This is a questions so I want an answer, there are no right or wrongs.

Reply:
we are sending category_id from client